src/client/app/helpers/strings.ts

```diff
@@ -52,7 +53,8 @@ export class Strings {
     static readonly githubLogoutFailed = 'Failed to sign out of GitHub';
     static readonly profileCheckFailed = 'Failed to get the GitHub profile';
     static readonly gistRetrieveFailed = 'Failed to retrieve GitHub gists';
-    static readonly gistShareFailed = 'Failed to share the GitHub gist';
+    static readonly gistShareFailedBody = 'Failed to share the GitHub gist';
+    static readonly gistShareFailedTitle = 'Share failed';
```

Comment:
Modified strings and added `/* NEEDS STRING REVIEW */` for all strings I've added to the file.